packages/common/src/utils/NumberUtil.ts

```diff
@@ -1,61 +1,68 @@
-import * as BigNumber from 'bignumber.js'
+type BigIntValue = string | number | bigint
 
 export const NumberUtil = {
-  bigNumber(value: BigNumber.BigNumber.Value) {
-    return new BigNumber.BigNumber(value)
+  bigNumber(value: BigIntValue) {
+    if (typeof value === 'bigint') {
+      return value
+    }
+
+    return toBigInt(value)
+  },
+
+  multiply(a: BigIntValue | undefined, b: BigIntValue | undefined) {
```

Comment:
Can we keep JSDocs? Function names quite plain but for some functions it's being helpful really